guides/release-drafter.md

```diff
@@ -0,0 +1,34 @@
+# Changelogs Automation: the Release-Drafter
+
+The [release-drafter](https://github.com/release-drafter/release-drafter/) tool is used in our GitHub Actions to automatize the content of each release description, so the changelogs.
+
+## Motivations
+
+We maintain a lot of packages and plugins than imply several releases we have to publish regularly.<br>
```

Comment:
```suggestion
We maintain a lot of packages and plugins implying several releases that we have to publish regularly.<br>
```

The goal was to correct the "than", so we can use something different than my suggestion, but still fix that :)